airflow/utils/logging.py

```diff
@@ -71,7 +71,7 @@ def read(self, remote_log_location, return_error=False):
             try:
                 s3_key = self.hook.get_key(remote_log_location)
                 if s3_key:
-                    return s3_key.get_contents_as_string().decode()
+                    return s3_key.get_contents_as_string()
```

Comment:
Yes... I'm not 100% sure that's the reason but usually extraneous decode()'s indicate Py2/3 compatibility issues